app/views/layouts/blacklight/base.html.erb

```diff
@@ -13,7 +13,7 @@
     <%= favicon_link_tag %>
     <%= stylesheet_link_tag "application", media: "all" %>
     <%= javascript_include_tag "application" %>
-    <%= javascript_pack_tag 'application' %>
+    <%#= Uncomment this to load the application pack javascript_pack_tag 'application' %>
```

Comment:
Since this partial is embedded in the gem, maybe we should just remove this file and provide instructions on how to add this in their local instances.